tools/indexer.py

```diff
@@ -21,6 +29,8 @@
 else:
   service_context = get_watsonx_context(model="ibm/granite-13b-chat-v1")
```

Comment:
shouldn't DEFAULT_MODEL be getting referenced somewhere?

Reply:
yes, good catch.